schema/PACK.xsd

```diff
@@ -17,14 +17,16 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 
-  $Date:        21. Apr 2022
-  $Revision:    1.7.7
+  $Date:        01.06.2022
```

Comment:
The date must be kept in the valid format
```suggestion
  $Date:        01. Jun 2022
```

Reply:
Done